rocAL/rocAL_pybind/examples/rocAL_api_tf_classification_reader.py

```diff
@@ -71,19 +71,18 @@ def main():
         )
         jpegs = inputs["image/encoded"]
         images = fn.decoders.image(jpegs, user_feature_key_map=feature_key_map, output_type=types.RGB, path=image_path)
-        resized = fn.resize(images, resize_x=300, resize_y=300)
+        resized = fn.resize(images, resize_width=300, resize_height=300)
         if(one_hot_labels == 1):
             labels = inputs["image/class/label"]
             _ = fn.one_hot(labels, num_classes=1000)
-        pipe.set_outputs(resized)
+        pipe.setOutputs(resized)
     # Build the pipeline
     pipe.build()
     # Dataloader
     image_iterator = ROCALIterator(pipe, device = device)
     cnt = 0
     # Enumerate over the Dataloader
     for i, (images_array, labels_array) in enumerate(image_iterator, 0):
```

Comment:
Make sure handle all the Dtypes, Layouts cases in 

def draw_patches(img, idx, device_type):